curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What command is used to see the path of working directory?
 
 #### --distractors--
 
-Placeholder distractor 1
+a) `ls`
 
 ---
 
-Placeholder distractor 2
+b) `cd`
 
 ---
 
-Placeholder distractor 3
+c) `pwd`
 
 #### --answer--
 
-Placeholder answer
+c) `pwd`
```

Comment:
there are 3 distractors and one answer, total 4 choices, and they need to be all different from each other.